SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialDispatcher.h

```diff
@@ -28,13 +27,40 @@ class SPATIALGDK_API USpatialDispatcher : public UObject
 	GENERATED_BODY()
 
 public:
+	using FCallbackId = uint32;
+
 	void Init(USpatialNetDriver* NetDriver);
 	void ProcessOps(Worker_OpList* OpList);
 
+	// AddOpCallback returns a callback ID which is incremented on each callback that is registered.
```

Comment:
Name in comment is out of date after renaming.

Reply:
🤦‍♂️